stable/coredns/templates/rbac.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if (.Values.middleware.kubernetes.enabled) and (.Values.rbac.create) }}
```

Comment:
This won't work. `and` is a function in Go templates. This should work:

`{{- if and .Values.middleware.kubernetes.enabled .Values.rbac.create }}`